s3/proto/s3.micro.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-micro. DO NOT EDIT.
-// source: s3/proto/s3.proto
+// source: s3.proto
```

Comment:
Hi @sunfch which version of protoc you used, seems the file is renamed.

Reply:
It show version 3.7.1